mock-echo-app/src/cmr/mock_echo/data/token_db.clj

```diff
@@ -35,7 +38,11 @@
   :id of the token that was created"
   [context token-info]
   (let [token-db (context->token-db context)
-        token (assoc token-info :id (new-token-id token-db))]
+        token (assoc token-info :id (new-token-id token-db))
+        token (if (:expires token)
+                token
+                ;; token expires in 30 days
+                (assoc token :expires (datetime-helper/datetime->string (t/plus (tk/now) (t/days 30)))))]
```

Comment:
We should have this be a defconfig

Reply:
This code is only for testing. We can make this a `def`, but probably not a `defconfig`.